### PR TITLE
[#147740351] Implemented OpenGraph tags for dojos

### PR DESCRIPTION
**Pull request by DanielBrierton**
(Yes, I'm aware I accidentally pushed this branch to CoderDojo instead of my own fork...)

### CODE REVIEW

File: index.ejs [hunk 1]
```diff
@@ -4,9 +4,17 @@
...
+        {{#each this}}
+          <meta name="og:{{@../key}}" content="{{this}}">
+        {{else}}
+          <meta name="og:{{@key}}" content="{{this}}">
+        {{/each}}
+      {{/each}}
+    <% } %>
     <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:300,400,700" type="text/css" />
     <script src="https://www.google.com/recaptcha/api.js?onload=vueRecaptchaApiLoaded&render=explicit" async defer></script>
-    <link rel="stylesheet" href="./assets/style.css" type="text/css" />
```
Comment (Wardormeur) [inline index.ejs]: Why was this removed?

File: index.js [hunk 2]
```diff
@@ -28,10 +33,17 @@ exports.register = function (server, options, next) {
 
   server.route({
     method: 'GET',
-    path: '/dojos/{country}/{path*}',
-    handler: {
-      file: {
-        path: __dirname + '/dist/index.html'
+    path: '/dojos/{id}/{alpha2*}',
```
Comment (Wardormeur) [inline index.js]: path changed, does that mean the default page url for dojos is also changed?

File: index.ejs (hunk 1 above)
Comment (DanielBrierton) [inline index.ejs]: The file doesn't exist, and has been resulting in a 404 since the start of the project 😅  It also caused issues with OpenGraph as it was a relative URL, and so Hapi picked it up as a Dojo URL

File: index.js (hunk 2 above)
Comment (Wardormeur) [inline index.js]: Nvmind, it's to match the preloader values am i right ?

Comment (DanielBrierton) [inline index.js]: The only thing that's looking at the names of these params is the dojo preloader, so I just changed it to match what it expects.

Comment (DanielBrierton) [inline index.js]: (So, basically, yes 😛 )
